### ISSUE TITLE
Accessibility (Table Block): Voice Control commands not recognized when using Table Block in mobile browser on iPad

### ISSUE THREAD

**Issue by pauljacobson** [WordPress/gutenberg#43251]
### Description

This is a report from a WordPress.com customer who explained as follows:

>I am using iPad iOS 15.6. I have tested and get same behavior on Chrome and Safari browsers.
>
>I am not able to enter or edit cells within a table block using Voice Control. This is an accessibility issue for me. I am currently not able to use tables in WordPress.

The customer's site does not support plugins.

### Step-by-step reproduction instructions

To replicate:

Turn on Voice Control
Create a table
Say show grid
Select the upper left-hand corner cell
Say hello world
Hello world will appear in that cell

Issue 1: The only two voice commands that appear to work are, delete and select line. None of the other Voice Control commands work for: dictation, text navigation, text selection, text editing, text deletion, (see full list of commands in settings).

Say show grid
Select the left hand cell second from the top
Say hello world
Hello world does not appear and that cell

Issue 2: Unable to enter text into cells using Voice Control.

### Screenshots, screen recording, code snippet

![IMG_8107_redacted](https://user-images.githubusercontent.com/2912202/184828038-87b4a32a-3d9c-4907-b67a-e61942d5b6e7.jpg)

![IMG_8105](https://user-images.githubusercontent.com/2912202/184828088-c2e0a502-a94d-454e-9aca-ee3e70d20aa7.jpg)

### Environment info

- Gutenberg: v13.8.2
- Editing Toolkit: 3.39221
- WP AMP: 2.0.5
- CoBlocks: 2.18.1-simple-rev.2
- Theme: Intergalactic 2
- Device OS: iPad OS 15.6
- Browsers: Chrome and Safari for iPad OS

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

No
